docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

```diff
@@ -84,9 +84,11 @@ The name should be just the name, without any additional info. For example, don'
 - Apostrophe (Special Edition)
   :::
 
+If the app is a distinct edition among different versions of your app on Flathub (e.g. `Plex` for desktop versus `Plex HTPC` for the home theater version), then using the full name including the edition may be acceptable.
+
 ### No weird formatting
 
-The name should not have any weird formatting or punctuation. For example, it should not be all-lowercase, all-uppercase, camel case, or contain dashes or periods. Cases where the weird formatting is part of the brand are exempt from this (e.g. `VLC`).
+The name should not have any weird formatting or punctuation. For example, it should not be all-lowercase, all-uppercase, camel case, pascal case, or contain dashes or periods. Cases where the formatting is part of an established brand may be exempt (e.g. `VLC`).
```

Comment:
To be explicit @bertob do we think `pascal case` is bad or is it just camel case?

Reply:
Any kind of weird formattng that's not on purpose, no strong opinion on what to include in the description. Personally I'd keep it simple here, I don't think most people, including developers, know pascal case...

Reply:
@bertob PascalCase vs camelCase; I'd say pascal case is more common in app names but yeah both are not really wanted. It's getting pedantic at that point; the gist is that ideally apps just have normal, word-like names. :)

Reply:
Right, i've seen both in the wild but the latter has somewhat better name recognition which is why I went with that. If it's not clear maybe we should just include examples for all the options rather than relying on these descriptions.